site/src/components/icons/index.ts

```diff
@@ -0,0 +1,14 @@
+export * from "./ApiKeyGenerated";
+export * from "./ArrowLeft";
+export * from "./ArrowRight";
```

Comment:
@benwerner01 Can you help rename these to contain *Icon* so that they are consistent with the others?

```suggestion
export * from "./ApiKeyGeneratedIcon";
export * from "./ArrowLeftIcon";
export * from "./ArrowRightIcon";
```

Reply:
Sure thing 👍

Reply:
Thanks!

Reply:
Addressed in https://github.com/blockprotocol/blockprotocol/pull/200/commits/4ef9ca12a335d0c255fbfadab072d4abeea15fb5